### ISSUE TITLE
Can you tell me some details about how to generate a random sparse matrix ?

### ISSUE THREAD

**Issue by yuanAIhan** [kokkos/kokkos-kernels#1117]
In /example/wiki/sparse/KokkosSparse_wiki_spgemm.cpp. Matrix multiplication is implemented here, but I am not very sure how to generate a random sparse matrix. Or I want to change the dimensional information of this sparse matrix. Can you tell me some details to figure it out?Because when I modify mat_structure(0, 0) = 100 and mat_structure(1, 0) = 100 , the code enters some infinite loops, and there is no result output.  
In short, I want to know how to generate this random sparse matrix.

**Comment by srajama1** [comment]
An internal example can be found here. This is just an example, because you asked for it. We do not plan to support this interface, hence it is in the Impl namespace.

https://github.com/kokkos/kokkos-kernels/blob/00189c0be23a70979aeaa162f0abd4c0e4d1c479/src/common/KokkosKernels_IOUtils.hpp#L70
